test/test_proxy_tensor.py

```diff
@@ -1584,34 +1575,13 @@ def f(args, kwargs, extra_args, extra_kwargs):
         args = [sample_input.input] + list(sample_input.args)
         kwargs = sample_input.kwargs
 
-        # If any argument is a torch.Size(), maybe get dynamic shapes for it by:
-        # - Create a temporary Tensor whose size is the torch.Size() we want. Note that
-        #   we use an expanded Tensor as we cannot pass "meta" Tensors to make_fx.
-        # - Pass it to make_fx such that it is is converted to a proxy Tensor
-        # - Unpack the size in the wrapper to get a torch.Size with dynamic shapes (in
-        #   symbolic mode, a no-op otherwise)
-        extra_args = []
-        extra_kwargs = {}
-        for i, arg in enumerate(args):
-            if isinstance(arg, torch.Size):
-                extra_args.append((i, torch.empty(arg, device="cpu")))
-        for key, value in kwargs.items():
-            if isinstance(value, torch.Size):
-                extra_kwargs[key] = torch.empty(value, device="cpu")
-
         try:
-            new_f = make_fx(f, tracing_mode=tracing_mode)(args, kwargs, extra_args, extra_kwargs)
-        except DynamicOutputShapeException as e:
+            optests.make_fx_check(fn, args, kwargs, tracing_mode, self.assertEqual)
+        except DynamicOutputShapeException:
             self.skipTest("Dynamic output shape operation in trace")
-        for arg in args:
-            if isinstance(arg, torch.Tensor) and arg.dtype == torch.float:
-                arg.uniform_(0, 1)
-        try:
-            old_out = f(args, kwargs, extra_args, extra_kwargs)
-        except Exception:
-            continue
-        new_out = wrapper_set_seed(new_f, args, kwargs, extra_args, extra_kwargs)
-        self.assertEqual(new_out, old_out)
+        except optests.TestFrameworkError:
+            return
```

Comment:
Does this mean that if we have opinfo example inputs that our test harness doesn't like, we'll auto-pass the test?

Reply:
If our test harness raises TestFrameworkError, then sure. It's not supposed to raise TestFrameworkError when it dislikes certain OpInfo example inputs. The exception here is to maintain the old behavior of try/except (see deleted lines)